hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/throttle/PressureAwareFlushThroughputController.java

```diff
@@ -79,6 +79,11 @@ protected void chore() {
       });
   }
 
+  @Override
+  public void updateConfig(Configuration conf, RegionServerServices server) {
```

Comment:
We do not need to implement the method for flush?

Reply:
> We do not need to implement the method for flush?

There is a empty updateConfig method in PressureAwareFlushThroughputController in this patch.
it can be Implemented in a new issue？